### PR TITLE
Various tweaks and fixes from the client

### PR DESCRIPTION
**Pull request by hammerfunk**
Lots of copy changes
Changed all title inputs from select to text inputs
Fixed uploads to have errors

### CODE REVIEW

File: app/assets/javascripts/application.js.coffee [hunk 1]
```diff
@@ -62,6 +62,9 @@ jQuery ->
         if $(this).closest(".question-block").find(".errors-container li").size() > 0
           $(this).closest(".question-block").find(".errors-container").empty()
       $(this).closest(".question-has-errors").removeClass("question-has-errors")
+  $(".supporters-list input").change () ->
```
Comment (matthewford) [inline app/assets/javascripts/application.js.coffee]: Empty parameter list is forbidden

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step1.rb [hunk 2]
```diff
@@ -84,7 +71,7 @@ def promotion_step1
           conditional :award_holder, :yes
         end
 
-        options :nominated_for_award, "Is the nominee currently being nominated for another award/honor?" do
+        options :nominated_for_award, "Is the nominee currently being nominated for another award/personal honour?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step1.rb]: Line is too long. [118/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb [hunk 3]
```diff
@@ -23,7 +23,7 @@ def promotion_step3
               What has their impact been? How wide is their influence? What are their achievements? Include details of the length of time involved and quantitative evidence that demonstrate the clear impact of the nominee’s activities.
             </p>
             <p>
-              Include when your nominee's activities have taken place within challenging circumstances or disadvantaged/deprived communities, or where young people or under-represented groups have benefited.
+              Include where your nominee's activities have taken place within challenging circumstances or disadvantaged/deprived communities, or where young people or under-represented groups have benefited.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step3.rb]: Line is too long. [208/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 4]
```diff
@@ -34,7 +34,7 @@ def promotion_step4
 
           hint "How does the letters of support system work?", %(
             <p>
-              First, you should enter details of your chosen supporters below. We will then email them a link to a web form, where they can enter their letter of support.
+              First, you should enter details of your chosen supporters below. We will then email them a link to a web form, where they can enter their letter of support and click on the ‘Send support request’ button
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [216/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 5]
```diff
@@ -64,7 +64,7 @@ def promotion_step4
               Letters of support should be about 500 words in length, making it clear why a nomination is being supported and NOT simply saying “I wish to support the nomination of …”.
             </p>
             <p>
-              The Queen’s Award for Enterprise Promotion recognises those who have gone above and beyond their duties promoting enterprise and entrepreneurial skills, so your letter should focus mainly on the nominee’s achievements outside of the requirements of their paid job, using your first hand knowledge of these where possible.
+              The Queen’s Award for Enterprise Promotion recognises individuals who go beyond the limits of their day-to-day role to foster an entrepreneurial spirit and promote enterprise within others, so your letter should focus mainly on the nominee’s achievements outside of the requirements of their paid job, using your first hand knowledge of these where possible.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [372/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 6]
```diff
@@ -107,7 +107,7 @@ def promotion_step4
         supporters :supporter_letters_list, "Manual upload" do
           context %(
             <p>
-              Here you can manually upload letters of support up to 5mb in size. This is for any of your supporters who would rather write a hard copy letter and send it to you directly.
+              Here you can manually upload letters of support each up to 5mb in size. This is for any of your supporters who would rather write a hard copy letter and send it to you directly.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [191/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 7]
```diff
@@ -152,6 +152,15 @@ def promotion_step4
             </p>
           )
 
+          hint "What are the allowed file formats?", %(
+            <p>
+              You can upload any of the following file formats:
+            </p>
+            <p>
+              chm, csv, diff, doc, docx, dot, dxf, eps, gif, gml, ics, jpg, kml, odp, ods, odt, pdf, png, ppt, pptx, ps, rdf, rtf, sch, txt, wsdl, xls, xlsm, xlsx, xlt, xml, xsd, xslt, zip
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [188/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step5.rb [hunk 8]
```diff
@@ -17,7 +17,15 @@ def promotion_step5
...
               We won't consider business plans, annual accounts or company policy documents. Additional materials should not be used as a substitute for completing sections of the form.
             </p>
-          ) # TODO!
+          )
+          hint "What are the allowed file formats?", %(
+            <p>
+              You can upload any of the following file formats:
+            </p>
+            <p>
+              chm, csv, diff, doc, docx, dot, dxf, eps, gif, gml, ics, jpg, kml, odp, ods, odt, pdf, png, ppt, pptx, ps, rdf, rtf, sch, txt, wsdl, xls, xlsm, xlsx, xlt, xml, xsd, xslt, zip
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step5.rb]: Line is too long. [188/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb [hunk 9]
```diff
@@ -52,7 +40,7 @@ def promotion_step6
 
         confirm :agree_being_contacted_about_issues_not_related_to_application, "Confirmation of contact" do
           ref "F 2"
-          text "I am happy to be contacted about Queen's Award for Enterprise issues not related to my application (e.g. acting as a case study, newsletters, other info)."
+          text "I am happy to be contacted about Queen's Awards for Enterprise issues not related to my application (e.g. acting as a case study, newsletters, other info)."
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb]: Line is too long. [172/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb [hunk 10]
```diff
@@ -64,7 +52,7 @@ def promotion_step6
           ref "F 3"
           required
           text %(
-            By ticking this box, I submit a nomination for consideration for the Queen’s Award for Enterprise Promotion 2015. I certify that all the particulars given, and those in any accompanying statements are correct to the best of my knowledge and belief and that no material information has been withheld.
+            By ticking this box, I submit a nomination for consideration for The Queen’s Award for Enterprise Promotion 2016. I certify that all the particulars given, and those in any accompanying statements are correct to the best of my knowledge and belief and that no material information has been withheld.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb]: Line is too long. [311/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 11]
```diff
@@ -118,7 +118,7 @@ def innovation_step1
           words_max 100
         end
 
-        options :other_awards_won, "Have you won any other business awards in the past?" do
+        options :other_awards_won, "Have you won any other business or enterprise awards in the past?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [105/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 12]
```diff
@@ -306,8 +306,16 @@ def innovation_step1
         upload :org_chart, "Upload an organisational chart." do
           ref "A 15"
           context %(
-            <p>You can submit files in all common formats, as long as they're less than 5mb.</p>
+            <p>You can submit files in all common formats, as long as they're less than 5mb each.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [101/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 13]
```diff
@@ -306,8 +306,16 @@ def innovation_step1
...
                     )
+          hint "What are the allowed file formats?", %(
+            <p>
+              You can upload any of the following file formats:
+            </p>
+            <p>
+              chm, csv, diff, doc, docx, dot, dxf, eps, gif, gml, ics, jpg, kml, odp, ods, odt, pdf, png, ppt, pptx, ps, rdf, rtf, sch, txt, wsdl, xls, xlsm, xlsx, xlt, xml, xsd, xslt, zip
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [188/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 14]
```diff
@@ -129,7 +129,7 @@ def innovation_step4
         confirm :declaration_of_corporate_responsibility, "" do
           required
           sub_ref "D 1.1"
-          text "I am not aware of any matter which might cast doubt on the worthiness of my organisation to receive a Queen's Award for Enterprise."
+          text "I am not aware of any matter which might cast doubt on the worthiness of my organisation to receive a Queen's Awards for Enterprise."
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [149/100]

File: app/forms/qae_2014_forms/innovation/innovation_step5.rb [hunk 15]
```diff
@@ -6,10 +6,18 @@ def innovation_step5
           ref "E 1"
           context %(
             <p>We can't guarantee these will be reviewed, so include any vital information within the form.</p>
-            <p>You can upload files in all common formats, as long as they're less than 5mb.</p>
+            <p>You can upload files in all common formats, as long as they're less than 5mb each.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step5.rb]: Line is too long. [101/100]

File: app/forms/qae_2014_forms/innovation/innovation_step5.rb [hunk 16]
```diff
@@ -6,10 +6,18 @@ def innovation_step5
...
             <p>You may link to videos, websites or other media you feel relevant.</p>
             <p>We won't consider business plans, annual accounts or company policy documents. Additional materials should not be used as a substitute for completing sections of the form.</p>
-                    ) # TODO!
+                    )
+          hint "What are the allowed file formats?", %(
+            <p>
+              You can upload any of the following file formats:
+            </p>
+            <p>
+              chm, csv, diff, doc, docx, dot, dxf, eps, gif, gml, ics, jpg, kml, odp, ods, odt, pdf, png, ppt, pptx, ps, rdf, rtf, sch, txt, wsdl, xls, xlsm, xlsx, xlt, xml, xsd, xslt, zip
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step5.rb]: Line is too long. [188/100]

File: app/forms/qae_2014_forms/innovation/innovation_step6.rb [hunk 17]
```diff
@@ -51,7 +39,7 @@ def innovation_step6
         confirm :agree_being_contacted_about_issues_not_related_to_application, "Confirmation of contact" do
           ref "F 3"
           text %{
-            I am happy to be contacted about Queen's Award for Enterprise issues not related to my application (e.g. acting as a case study, newsletters, other info).
+            I am happy to be contacted about Queen's Awards for Enterprise issues not related to my application (e.g. acting as a case study, newsletters, other info).
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step6.rb]: Line is too long. [167/100]

File: app/forms/qae_2014_forms/innovation/innovation_step6.rb [hunk 18]
```diff
@@ -66,10 +54,10 @@ def innovation_step6
           ref "F 4"
           required
           text %(
-            By ticking this box, I certify that all the particulars given and those in any accompanying statements are correct to the best of my knowledge and belief and that no material information has been withheld. I undertake to notify The Queen’s Awards Office of any changes to the information I have provided in this entry form.
+            By ticking this box, I submit an entry for consideration for The Queen’s Awards for Enterprise 2016. I certify that all the particulars given and those in any accompanying statements are correct to the best of my knowledge and belief and that no material information has been withheld. I undertake to notify The Queen’s Awards Office of any changes to the information I have provided in this entry form.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step6.rb]: Line is too long. [415/100]

File: app/forms/qae_2014_forms/innovation/innovation_step6.rb [hunk 19]
```diff
@@ -66,10 +54,10 @@ def innovation_step6
...
             <br>
             <br>
-            I am not aware of any matter which might cast doubt upon the worthiness of this business unit to receive a Queen’s Award. I consent to all necessary enquiries being made by The Queen’s Awards Office in relation to this entry. This includes enquiries made of Government Departments and Agencies in discharging its responsibilities to vet any business unit which might be granted a Queen’s Award to ensure the highest standards of propriety.
+            I am not aware of any matter which might cast doubt on the worthiness of my organisation to receive a Queen's Award for Enterprise. I consent to all necessary enquiries being made by The Queen’s Awards Office in relation to this entry. This includes enquiries made of Government Departments and Agencies in discharging its responsibilities to vet any business unit which might be granted a Queen’s Award to ensure the highest standards of propriety.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step6.rb]: Line is too long. [461/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 20]
```diff
@@ -116,7 +116,7 @@ def trade_step1
           words_max 100
         end
 
-        options :other_awards_won, "Have you won any other business awards in the past?" do
+        options :other_awards_won, "Have you won any other business or enterprise awards in the past?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [105/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 21]
```diff
@@ -252,8 +252,16 @@ def trade_step1
         upload :org_chart, "Upload an organisational chart." do
           ref "A 16"
           context %(
-            <p>You can submit files in all common formats, as long as they're less than 5mb.</p>
+            <p>You can submit files in all common formats, as long as they're less than 5mb each.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [101/100]

File: app/pdf_generators/qae_pdf_forms/general/draw_elements.rb [hunk 22]
```diff
@@ -110,7 +110,7 @@ def render_urn
   end
 
   def render_award_information
-    text_box form_answer.decorate.award_application_title.upcase,
+    text_box "Queen’s Award for Enterprise Promotion #{Date.today.year}".upcase,
```
Comment (RuslanHamidullin) [inline app/pdf_generators/qae_pdf_forms/general/draw_elements.rb]: @hammerfunk small note:
need to replace:
```
"Queen’s Award for Enterprise Promotion #{Date.today.year}".upcase
```
with:
```
"Queen’s Award for Enterprise Promotion #{AwardYear.current.year}".upcase
```

**Comment by RuslanHamidullin** [comment]
All good for me :+1:

**Pull request merged**